packages/search-token-server/app.ts

```diff
@@ -7,7 +7,7 @@ const app = express();
 
 app.use(express.json());
 
-app.get<{}, any, {token: string}>(
+app.get<Record<string, never>, any, {token: string}>(
```

Comment:
Maybe: `Record<string, unknown>` ?

Reply:
`any` would work, but not `unknown` because of this condition: `Type 'unknown' is not assignable to type 'string'.`

Reply:
now that server/ has its own estlint, I will revert to the original type